playbooks/webapp/README.md

```diff
@@ -157,6 +157,26 @@ To delete the webapp:
   ```
 * **deploy_flask_app_force_init** (bool): Whether to drop existing tables and create new ones when deploying the webapp database. Default: `false`
 
+### webapp deployment in HA architecture
+
+`webapp_ha_aurora.yaml` playbook deploys the flask app to a cross region high availability architecture. The playbook replicates the app deployment to a second region. The backend is an Aurora global cluster. For adding the write forwarding feature, aurora-mysql can be used. Default db engine is aurora-postgresql. The app in each region is configured to access the associated Aurora cluster. In front of the two regions, route53 records are added to provide cross region DNS (failover scenario).
+
+Along with the above variables, following variables are need for this playbook:
+
+* **test_instance_class** (str): DB instance class for the aurora db instances. Default: `db.r5.large`
```

Comment:
is there a reason to prefix all vars with `test_` ? it sounds like testing variables
It seems like `resource_prefix` is also a playbook variable

Reply:
I will address this.